llvm/tools/llvm-profgen/PerfReader.cpp

```diff
@@ -41,6 +41,17 @@ static cl::opt<bool>
                                 "and produce context-insensitive profile."));
 cl::opt<bool> ShowDetailedWarning("show-detailed-warning",
                                   cl::desc("Show detailed warning message."));
+cl::opt<bool>
+    LeadingIPOnly("leading-ip-only",
+                  cl::desc("Form a profile based only on sample IPs"));
+
+static cl::list<std::string> PerfEventFilter(
+    "perf-event",
+    cl::desc("Ignore samples not matching the given event names"));
```

Comment:
nit: name it `profiled-event` with description `Perf event to generate profile for, e.g. br_misp_retired.all_branches. Program execution count profile is generated when unspecified.`